tests/testthat/test-release.R

```diff
@@ -130,12 +130,18 @@ test_that("returns empty string if no bullets", {
 test_that("can find milestone numbers", {
   skip_if_offline("github.com")
 
+  tr <- list(
```

Comment:
I think it would be worth adding a test that `gh_milestone_number()` returns `NA` when `gh()` errors (e.g. because you've mocked it to always error)

Reply:
done